content/applications/inventory_and_mrp/purchase/manage_deals/calls_for_tenders.rst

```diff
@@ -1,267 +1,285 @@
-quantity in the :guilabel:`Quantity` column, and change the purchase price in the
-:guilabel:`Unit Price` column, if desired.
+Begin with creating a new |RfQ|, by navigating to :menuselection:`Purchase app --> New`. Fill out
+the |RfQ| according to the earlier instructions.
 
-Once ready, click :guilabel:`Send by Email`. This causes a :guilabel:`Compose Email` pop-up window
-to appear, wherein the message to the vendor can be customized. Once ready, click :guilabel:`Send`
-to send an email to the vendor listed on the purchase order form.
+Then, once ready, click the :guilabel:`Alternatives` tab. Since this new |PO| was created
+separately, there are no other orders linked yet.
 
-Then, click the :guilabel:`Alternatives` tab once more. Since this new :abbr:`PO (Purchase Order)`
-was created separately, there are no other orders linked yet. To link this order with the
-alternatives created previously, click :guilabel:`Link to Existing RfQ` on the first line in the
-:guilabel:`Vendor` column.
+To link this order with alternatives, click :guilabel:`Link to Existing RfQ` on the first line in
+the :guilabel:`Vendor` column.
 
-.. image:: calls_for_tenders/calls-for-tenders-link-existing-rfq.png
+.. image:: calls_for_tenders/calls-for-tenders-link-rfq-popup.png
    :align: center
-   :alt: pop-up to link new quotation to existing RFQs.
+   :alt: Pop-up to link new quotation to existing RFQs.
 
-This causes an :guilabel:`Add: Alternative POs` pop-up window to appear. Select the three purchase
-orders created previously, and click :guilabel:`Select`. All of these orders are now copied to this
-:abbr:`PO (Purchase Order)` under the :guilabel:`Alternatives` tab.
+This opens an :guilabel:`Add: Alternative POs` pop-up window. Select the three previously-created
+|POs|, and click :guilabel:`Select`. All of these orders are now copied to this |PO|, and can be
+found under the :guilabel:`Alternatives` tab.
 
 .. tip::
-   If a large number of purchase orders are being processed and the previous
-   :abbr:`POs (Purchase Orders)` can't be located, try clicking :menuselection:`Group By -->
-   Vendor` under the search bar at the top of the pop-up window to group by the vendors selected on
-   the previous orders.
+   If a large number of |POs| are being processed and the previous |POs| can't be located, click
+   :menuselection:`Group By --> Vendor` (under the search bar, at the top of the pop-up window) to
+   group the vendors selected on the previous orders.
 
 Compare product lines
 =====================
 
-When there are multiple :abbr:`RFQs (Requests for Quotation)` linked as alternatives, they can be
-compared side-by-side in order to determine which vendors offer the best deals on which products.
-To compare each quotation, go to the :menuselection:`Purchase` app, and select one of the
-quotations created previously.
+Alternative |RfQs| can be compared side-by-side in order to determine which vendors offer the best
```

Comment:
```suggestion
Alternative |RfQs| can be compared side-by-side, in order to determine which vendors offer the best
```
I agree that RfQs should be used like this (as it seems the most appropriate), but the lack of common usage throughout the doc may make this doc hard to follow for some readers, so try to make sure the most accurate terms are used uniformly throughout, please. Thanks! 👍